.github/workflows/build.yml

```diff
@@ -262,27 +261,27 @@ jobs:
           # https://github.com/opencontainers/image-spec/blob/master/annotations.md
           labels: "\
             org.opencontainers.image.created=${{
-              needs.prepare.outputs.created }}
+            needs.prepare.outputs.created }}
 
             org.opencontainers.image.description=${{
-              fromJson(needs.prepare.outputs.repometa).description }}
+            fromJson(needs.prepare.outputs.repometa).description }}
 
             org.opencontainers.image.licenses=${{
-              fromJson(needs.prepare.outputs.repometa).license.spdx_id }}
+            fromJson(needs.prepare.outputs.repometa).license.spdx_id }}
 
             org.opencontainers.image.revision=${{ github.sha }}
 
             org.opencontainers.image.source=${{
-              fromJson(needs.prepare.outputs.repometa).clone_url }}
+            fromJson(needs.prepare.outputs.repometa).clone_url }}
 
             org.opencontainers.image.title=${{
-              fromJson(needs.prepare.outputs.repometa).name }}
+            fromJson(needs.prepare.outputs.repometa).name }}
 
             org.opencontainers.image.url=${{
-              fromJson(needs.prepare.outputs.repometa).html_url }}
+            fromJson(needs.prepare.outputs.repometa).html_url }}
 
             org.opencontainers.image.version=${{
-              needs.prepare.outputs.source_version }}"
+            needs.prepare.outputs.source_version }}"
```

Comment:
Do not make unnecessary changes to the skeleton code.  This path leads to [The Tree of Woe](https://conan.fandom.com/wiki/Tree_of_Woe).

Reply:
Sorry about that. I removed this earlier and auto formatter must have added it back. i'll remove it again shortly.

Reply:
@jsf9k resolved with [06bd355](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/06bd355907f2ed9b9f602b6b3ab27dbb1db394bf)